gluonnlp/model/highway.py

```diff
@@ -0,0 +1,108 @@
+# coding: utf-8
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Highway layer."""
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+__all__ = ['Highway']
+
+from mxnet import gluon
+from mxnet.gluon import nn
+
+
+class Highway(gluon.Block):
+    r"""Highway network.
+
+    We implemented the highway network proposed in the following work::
+
+        @article{srivastava2015highway,
```

Comment:
It seems arXiv:1505.00387 is considered by the authors as extended abstract whereas _Srivastava, R. K., Greff, K., & Schmidhuber, J. (2015). Training very deep networks. In Advances in neural information processing systems (pp. 2377-2385)._ is the full paper. Should the latter be linked / cited as well?

Reply:
Yes, let's cite both of them. Interestingly, the shorter one has more citations...